### ISSUE TITLE
Feature request: use task template data in task

### ISSUE THREAD

**Issue by bubje** [jrial/fosdem-volunteers#52]
It would be handy if data in the task template could be automatically loaded in a task once you select the task template. That way one could either use that task template provided data or make some changes to it (or remove it) and thus have customer data. If the task template data only gets added to the task at time of the creation of the task, one would be able to do this.

One might need to look first at what the 'task template' is currently used for.

**Comment by jrial** [comment]
No, the idea is that by default it shows the description of the template, followed by the one from the task. That way, the template can contain the generic info, and the task optionally the task-specific info.

The reason this hasn't happened yet is because currently the data is a bit of a mess. Descriptions are duplicated across tasks and templates. Need to go over that, then it can be fixed. The fix itself is actually not that big, it's just that currently it's quite a chore to clean up the data due to the limitations of the admin interface and the size of our task list.

But I don't see why it wouldn't be possible to fix this with one SQL query, so eventually I might get a round tuit. ;)
